README.md

```diff
@@ -40,32 +39,51 @@ runner pod and engine monitor service), which are created & managed by it in ord
 The ChaosEngine is the core schema that defines the chaos workflow for a given application. Currently, it defines the following:
 
 - Application Data (namespace, labels, kind)
-- List of Chaos Experiments to be executed
-- Attributes of the experiments, such as, rank/priority 
-- Execution Schedule for the batch run of the experiments
+- ServiceAccount used for execution of the experiment
+- Chaos Experiment to be executed on the application
+- Attributes of the experiments (overrides defaults specified in the experiment CRs)
+- Chaos resource cleanup Option 
 
 The ChaosEngine is the referenced as the owner of the secondary (reconcile) resource with Kubernetes deletePropagation 
 ensuring these also are removed upon deletion of the ChaosEngine CR.
 
 Here is a sample ChaosEngineSpec for reference: 
 
-  ```yaml
-  apiVersion: litmuschaos.io/v1alpha1
-  kind: ChaosEngine
-  metadata:
-    name: engine-nginx
-  spec:
-    appinfo: 
-      appns: default
-      applabel: "app=nginx"
-    experiments:
-      - name: pod-delete 
-        spec:
-          rank: 
-      - name: container-kill
-        spec:
-          rank:  
-  ```
+```yaml
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: nginx-chaos
+  namespace: default
+spec:
+  appinfo:
+    appns: 'default'
+    applabel: 'app=nginx'
+    appkind: 'deployment'
+  annotationCheck: 'true'  
+  auxiliaryAppInfo: ''
+  chaosServiceAccount: nginx-sa
+  monitoring: false
+  components:
+    runner:
+      image: 'litmuschaos/chaos-executor:1.0.0'
+      type: 'go'
+  jobCleanUpPolicy: 'delete' 
+  experiments:
+    - name: pod-delete
+      spec:
+        components:
+          env:
```

Comment:
env field is not present in `1.0.0 release`. Is it okay to use?

Reply:
considering we are in master, it should be fine. But taking into account users may copy from here, reverted to previous schema and added note to say (as of 1.0.0)